backend/src/main/resources/application.yaml

```diff
@@ -50,24 +50,9 @@ spring:
       initialize-schema: never
       table-name: ${spring.jpa.properties.hibernate.default_schema}.spring_session
 management:
-  health.probes.enabled: true
-graphql:
-  servlet:
-    mapping: /graphql
-    enabled: true
-    corsEnabled: true
-    exception-handlers-enabled: true
-  playground:
-    enabled: false
```

Comment:
Can confirm that when running locally, the playground is still disabled with the new library. I wasn't able to get it up in dev2 either, so I assume it's off by default